R/html-tweak.R

```diff
@@ -242,8 +242,8 @@ tweak_homepage_html <- function(html, strip_header = FALSE) {
 
 # Mutates `html`, removing the badge container
 badges_extract <- function(html) {
-  # First try specially named div; then try first paragraph
-  x <- xml2::xml_find_first(html, "//div[@id='badges']")
+  # First try usethis-readme-like paragraph; then try first paragraph
```

Comment:
I think we should continue looking for the specially named div, unless you're sure no one ever used it.

Reply:
Right :woman_facepalming: Should it also be documented or rather not?

Reply:
No, can just leave as is.